src/pages/project/project_wizard/ProjectManager.tsx

```diff
@@ -60,7 +53,7 @@ export const ProjectManager = ({ suggestedProjectManager }: ProjectContributorsP
                   contributorIndex={projectManagerIndex}
                   baseFieldName={`${name}[${projectManagerIndex}]`}
                   contributor={projectManager}
-                  removeContributor={() => removeProjectManager(name, remove)}
+                  removeContributor={() => onRemoveProjectManager(name)}
```

Comment:
finnes et argument for å ha `onRemoveProjectManager` inline her i stedet for en egen funksjon. Da ville det blitt opprettet en funkson i stedet for to, men først og fremst at det hadde vært tydligere hva som skjer. Når jeg så `const onRemoveProjectManager = (name: string) => (...)` tenkte jeg at den tar inn navnet på prosjektlederen som skal fjernes, som virket shady